docs/source/release_notes.rst

```diff
@@ -6,9 +6,16 @@ Release Notes
         * Updated ``make_pipeline`` to not add ``Imputer`` if input data does not have numeric or categorical columns :pr:`1967`
     * Fixes
     * Changes
+        * Removed ``data_checks`` parameter, ``data_check_results`` and data checks logic from ``AutoMLSearch`` :pr:`1935`
     * Documentation Changes
     * Testing Changes
 
+.. warning::
+
+    **Breaking Changes**
+        * Removed ``data_checks`` parameter, ``data_check_results`` and data checks logic from ``AutoMLSearch`` To run data checks, please call DataCheck.validate(X_train, y_train) or take a look at our documentation for more examples. :pr:`1935`
```

Comment:
Could we say "To run the data checks which were previously run by default in `AutoMLSearch`, please call `DefaultDataChecks().validate(X_train, y_train) or" etc?